### PR TITLE
use req for sweepRequest

### PR DESCRIPTION
**Pull request by ubbabeck**
This pr adds the following changes.

- [039247c](https://github.com/breez/breez-sdk/pull/603/commits/039247c4db3ad239a14dd67cb1e70d609c58033d) Which requires `PrepareSweepRequest` as parameter.
- [ba6493b](https://github.com/breez/breez-sdk/pull/603/commits/ba6493bd6b772df156ec13ab932a1cbbbf4f31c9) Changes name of param to `sats_per_vbyte` for consistency for both `Sweep` and `Refund` request and prepare . 

Not sure if `sats_per_vbyte` is the preferred name, but I think we should use the same name for consistency.

### CODE REVIEW

File: libs/sdk-bindings/src/breez_sdk.udl [hunk 1]
```diff
@@ -678,7 +678,7 @@ dictionary PrepareRefundResponse {
 dictionary RefundRequest {
     string swap_address;
     string to_address;
-    u32 sat_per_vbyte;
+    u32 sats_per_vbyte;
```
Comment (JssDWt) [inline libs/sdk-bindings/src/breez_sdk.udl:681]: `sat_per_vbyte` is probably fine. Not worth a breaking change on the `RefundRequest` and the `PrepareRefundRequest` imo.

**Review by JssDWt — COMMENTED** [review]
One comment about breaking change. Otherwise looks good.

File: libs/sdk-bindings/src/breez_sdk.udl (hunk 1 above)
Comment (ubbabeck) [reply libs/sdk-bindings/src/breez_sdk.udl]: Good point. Fixed

**Review by JssDWt — APPROVED** [review]
LGTM!

**Pull request merged**